src/components/Picker/index.js

```diff
@@ -14,22 +15,40 @@ const propTypes = {
     /** Should the picker appear disabled? */
     isDisabled: PropTypes.bool,
 
-    /** Should the input be styled for errors  */
-    hasError: PropTypes.bool,
+    /** Input value */
+    value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
 
     /** Error text to display */
     errorText: PropTypes.string,
 
     /** Customize the Picker container */
     containerStyles: PropTypes.arrayOf(PropTypes.object),
+
+    /** Indicates that the input is being used with the Form component */
+    isFormInput: PropTypes.bool,
+
+    /**
+     * The ID used to uniquely identify the input
+     *
+     * @param {Object} props - props passed to the input
+     * @returns {Object} - returns an Error object if isFormInput is supplied but inputID is falsey or not a string
+     */
+    inputID: props => FormUtils.getInputIDPropTypes(props),
```

Comment:
This function has been renamed
```suggestion
    inputID: props => FormUtils.validateInputIDProps(props),
```